index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta content="width=device-width,initial-scale=1" name="viewport">
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="styles/base.css">
+  <title> DWBeyer</title>
+
+  <!-- handlebars template -->
+  <script id="projects-template" type="text/x-handlebars-template">
+    <article data-category="{{techUsed}}">
+      <header>
+        <h1><a href="{{link}}">{{title}}</a></h1>
+        <div class="projInfo">
+          Published: {{publishedOn}}{{publishStatus}}<br>
+          Technology Used: {{techUsed}}
+        </div>
+      </header>
+      <section class="projSummary">
+        <img class="tinted-image" src={{image}}>
```

Comment:
Using tbd as the img src doesn't really work and is giving you the broken image icon.
Since you may or may not have an image, you could use the handlebars if/else built-in helper to conditionally render an `<img>` or a `<div>` if there is no image.
Then you'd have to remove the `image:  'tbd'` from your data.

Like this:
```
{{#if image}}
  <img class="tinted-image" src={{image}}>
{{else}}
  <div class="tinted-image"></div>
{{/if}}
```

Read more about handlebars built-in helpers at http://handlebarsjs.com/builtin_helpers.html.

Reply:
already updated. it was a temporary placeholder until the image was added - not for production.